pkg/tuple/tuple.go

```diff
@@ -198,8 +198,11 @@ func StringToUserProto(userKey UserString) *openfgav1.User {
 	}}}
 }
 
-// SplitObject splits an object into an objectType and an objectID. If no type is present, it returns the empty string
-// and the original object.
+// SplitObject splits an object into an objectType and an objectID.
+// E.g.
+//  1. "group:fga" returns [group, fga].
```

Comment:
fwiw seeing square brackets here made me think the function returned a slice/array.

Reply:
updated